src/main/resources/test_files/runtime/FunctionObjectRemoveKeys.iq

```diff
@@ -0,0 +1,2 @@
+(:JIQS: ShouldRun; Output="( { "Engineer" : "Scott" } )" :)
+let $o := {"Captain" : "Kirk", "First Officer" : "Spock", "Engineer" : "Scott"} return remove-keys($o, ("Captain", "First Officer"))
```

Comment:
We'd need a test that remove non-existing keys, and another that removes a mix of existing and non-existing keys.